src/services/auth/plugins/password/service.ts

```diff
@@ -170,42 +168,32 @@ export class MemberPasswordService {
     return memberRepository.get(id);
   }
 
-  /**
-   *
-   * @param actor
-   * @param repositories
-   * @param body
-   * @param challenge  used for mobile
-   * @returns
+  /** Authenticate a member with email and password.
+   * @param repositories Repositories needed to interact with the database. Must contain a memberRepository and a memberPasswordRepository.
+   * @param email The email of the member that wants to authenticate.
+   * @param password The password of the member that wants to authenticate.
+   * @returns The member if the credentials are correct. Otherwise, undefined.
+   * @throws MemberNotSignedUp if the email is not registered.
+   * @throws MemberWithoutPassword if the member doesn't have a password.
    */
-  async login(
-    actor: undefined,
+  async authenticate(
     repositories: Repositories,
```

Comment:
Unfortunately, I would have to modify the parameters of `memberPasswordService.authenticate(...)` which I'm okay with that, but it would be almost the only function to have specific repositories need in parameter.